source/portable/NetworkInterface/ATSAME5x/NetworkInterface.c

```diff
@@ -59,15 +59,21 @@
 
 /* Check for optimal performance parameters */
 #if ( CONF_GMAC_NCFGR_RXCOEN == 0 )
-    #warning This driver works best with RX CRC offloading enabled.
+    #if ( ipconfigPORT_SUPPRESS_WARNING != 0 )
```

Comment:
Shouldn't the checks be `#if ( ipconfigPORT_SUPPRESS_WARNING == 0 )`

Reply:
This configuration is used to "suppress" the warning messages. So we should skip warning users when it's 1.

Reply:
Yes, but `#if ( ipconfigPORT_SUPPRESS_WARNING != 0 )` won't skip the warning when `ipconfigPORT_SUPPRESS_WARNING` is set to 1.

Reply:
Oops. My fault. Let me update it once again.